<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudPlatillo</class>
 <widget class="QWidget" name="CrudPlatillo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CrudPlatillo{
background: #C8CFE1;
}</string>
  </property>
  <widget class="QLabel" name="imagen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>361</width>
     <height>167</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>344</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10px &quot;SF Pro Display&quot;;

color:#3E5066;</string>
   </property>
   <property name="text">
    <string>Selecciona un platillo</string>
   </property>
  </widget>
  <widget class="QTableView" name="tV_categorias">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
color: #3C6AF1;</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 4px;
background-color: #FFFFFF;</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="LayoutCategoria">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9px &quot;SF Pro Text&quot;;
color: #3E5066;</string>
   </property>
   <property name="text">
    <string>*Doble click en una categoría para agregarla.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descripcionPlatillo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #C8CFE1;

font: 15px &quot;SF Pro Display&quot;;
color:#3E5066;
font-size: 10px;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SF Pro Display'; font-size:10px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SF Pro Text'; font-size:11pt;&quot;&gt;Descripción del platillo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_EliminarPlatillo">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1.35516px solid #3E5066;
border-radius: 9.03438px;
/*background-color: #C8CFE1;*/
background-color: none;


font: 14.9368px &quot;SF Pro Text&quot;;
color:#3E5066;
height: 38.85px;

margin-right: 5px;</string>
      </property>
      <property name="text">
       <string>Eliminar platillo</string>
      </property>
      <property name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/Img/EliminarPlatillo.png</normaloff>:/Img/EliminarPlatillo.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_guardarCambios">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 9.03438px;
background-color: #3E5066;

font: 14.9368px &quot;SF Pro Text&quot;;
color:#A9AFC2;
height: 38.85px;

margin-left: 5px;</string>
      </property>
      <property name="text">
       <string>Guardar cambios</string>
      </property>
      <property name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/Img/GuardarCambios.png</normaloff>:/Img/GuardarCambios.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btn_agregarCategoria">
      <property name="styleSheet">
       <string notr="true">border-radius: 4px;
background-color: #FFFFFF;
width: 100px;
height: 17.85px;

font: 14.9368px &quot;SF Pro Text&quot;;
color: #3C6AF1;</string>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
      <property name="icon">
       <iconset resource="../Resources.qrc">
        <normaloff>:/Img/AgregarCategoria.png</normaloff>:/Img/AgregarCategoria.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lE_categoria">
      <property name="styleSheet">
       <string notr="true">background: #FFFFFF;

border: 1px solid #C8CFE1;
border-radius: 4px;

color: #3C6AF1;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #C8CFE1;

font: 9px &quot;SF Pro Text&quot;;

color:#3E5066;</string>
   </property>
   <property name="text">
    <string>CATEGORÍAS:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombrePlatillo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #C8CFE1;

font: 20px &quot;SF Pro Display&quot;;

color:#3E5066;
</string>
   </property>
   <property name="text">
    <string>Nombre del platillo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agregarImagen">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
background-color: #FFFFFF;
width: 44px;
height: 44px;




</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/Img/EditarFotoPlatillo.png</normaloff>:/Img/EditarFotoPlatillo.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="precioPlatillo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>106</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #C8CFE1;

font: 15x &quot;SF Pro Display&quot;;
color:#3E5066;

</string>
   </property>
   <property name="text">
    <string>Precio $$$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>16</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #C8CFE1;

font: 15px &quot;SF Pro Display&quot;;
color:#3E5066;
</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
